_sass/_custom.scss

```
@@ -0,0 +1,81 @@
+// Bootstrap overrides
+//
+// Copy variables from `bootstrap/_variables.scss` to this file to override default values
+// without modifying source files.
+
+// Table of Contents
+//
+// Colors
+// Body
+// Typography
+// Tables
+// Buttons
+// Navbar
+// Breadcrumbs
+// Code
+
+// Colors
```

```suggestion
// Colours
```

okay, i am really excited about this part below as we are going to be getting our hands dirty

---

just doing some general management tasks waiting for my coffee to kick in (we gonna tackle this one in about 30 min)

---

sorry, for delay &mdash; first time creating account there:

- https://www.colourlovers.com/lover/DerekNonGeneric

---

> best thing to do now would be to first get all those colors

the goal here is simply to be able to get a more clear mental picture of what we have going on and what we are missing; we are just taking inventory, but eventual color corrections are to be expected [#colorTheory](https://twitter.com/search?q=%23colorTheory&src=typed_query)

---

the strategy we pulled off last night was refreshing, got my creative juices flowing, and was an exercise worth repeating, so expect an increase to the 1/6<sup>th</sup> we made in progress you'll find on ColourLovers profile

> we're going to be tackling 👆 in ~30min
> 
> just doing some general management tasks waiting for my coffee to kick in ❗
> 
> we seem to have 3 palettes necessary, but each of those would need color schemes for both light and dark modes, which would result in 6 distinct palettes ❗
> 
>     * https://www.colourlovers.com/palettes/add
>
> _Originally posted by @DerekNonGeneric in https://github.com/OpenINF/openinf.github.io/pull/1100#discussion_r1468773883_